### DISCUSSION TITLE
Change the text displayed when mouse hover the home's profile picture avatar

### DISCUSSION THREAD

**Discussion by ThierryLangie** [hugo-fixit/FixIt#345] [Q&A] [answered]
I would like to change the text displayed when the mouse hover the home's profile picture. For this moment, on my website, the path to the image is displayed in the tooltip. On the website of the author of this theme, his personal name is displayed in the tooltip. I searched in the configuration files (*.toml), tried a lot of things without success.

![image](https://github.com/hugo-fixit/FixIt/assets/13705487/b2d1caef-5107-4042-b93c-6ab60f084006)

![image](https://github.com/hugo-fixit/FixIt/assets/13705487/4b6b09e3-d2d8-46d3-b7d9-1f705e349b16)

**Comment by Lruihao** [comment, marked as answer]
you should set the `$.Site.Author.name`, like this [config](https://github.com/hugo-fixit/docs/blob/main/config/_default/author.toml).

**Reply by ThierryLangie** [reply]
First, thanks for your really quick response :) and your theme is amazing !

I did as showed on the picture below but still the same problem. I'm sure I did something wrong but I don't know what. 

So 
- I created a new folder called `config` inside of the theme's folder
- I created a new file inside this folder called `author.toml`
- I copy/paste the config inside of this file
- restarted the server

PS: I tried to put this `author.toml` file in the same folder as the `config.toml` file (...\themes\FixIt\...) but nothing changed

![image](https://github.com/hugo-fixit/FixIt/assets/13705487/f7d4c677-27b4-4bd7-88f1-113e816d653f)

**Reply by Lruihao** [reply]
> ![image](https://github.com/hugo-fixit/FixIt/assets/13705487/f7d4c677-27b4-4bd7-88f1-113e816d653f)

:) thanks for your using the FixIt as also！

and, if your configuration is all aggregated in one file name `config.toml`, you only need to add the following code to that file.

```
# Author config
[author]
  name = "xxxx"
  email = ""
  link = ""
  avatar = "/images/avatar.png"
```

btw, you could head to this [page](https://fixit.lruihao.cn/documentation/basics/#theme-configuration) for more config.

**Reply by ThierryLangie** [reply]
Okay so, first, I added these `author` parameters inside of `config.toml` (see below). And restarted the server. Didn't worked.

Next, I added these `author` parameters inside of `hugo.toml` (see below). No need to restart, the server detected the changes. Worked :)

I don't know why did I need to put these parameters inside of `hugo.toml` and not `config.toml`... I'm still novice with Hugo so excuse my ignorance.

![image](https://github.com/hugo-fixit/FixIt/assets/13705487/13b0ae95-1c41-40d4-a3fd-0305e81e538c)

![image](https://github.com/hugo-fixit/FixIt/assets/13705487/bb5cea54-260a-4a2a-9313-de70b9cbfe5e)

**Reply by Lruihao** [reply]
oh，i guess maybe Hugo change the default configuration file from config.toml to hugo.toml in high versions, so your configuration only works in hugo.toml.

**Reply by ThierryLangie** [reply]
Your guess is right. Thanks for your time. It works pretty well right now.
